src/helpers/strings-es.ts

```diff
@@ -1,56 +1,61 @@
-	"external-account-reconnect": "Volver a conectar",
-	"external-account-refresh": "Actualizar",
-	"external-account-remove": "Eliminar",
-	"device-data-no-data": "Si ha conectado Apple Health, Google Fit o Fitbit, vuelva más tarde para ver sus datos.",
-	"no-notifications-received": "No se recibieron notificaciones."
+    "back": "Atrás",
+    "close": "Cerrar",
+    "connect-ehr-title-prefix": "Conectar ",
+    "connect-ehr-title-divider": " o ",
+    "connect-ehr-title-providers": "proveedor",
+    "connect-ehr-title-health-plans": "plan de salud",
+    "connect-ehr-connected": "¡Estamos recibiendo su información de EHR!",
+    "connect-ehr-needs-attention": "Una de tus cuentas requiere atención.",
+    "connect-ehr-text": "Sus registros electrónicos de salud son una fuente de información importante. Con ellos, usted podría ayudar a los investigadores de la salud a realizar nuevos descubrimientos. Conecte a su proveedor o a su plan de salud (por ejemplo: Medicare) con @@PROJECT_NAME@@.",
+    "connect-ehr-text-connected": "Gracias por compartir tus datos de EHR con @@PROJECT_NAME@@. Conecta a otro proveedor para compartir más datos o administrar tus conexiones de EHR.",
+    "connect-ehr-not-enabled": "EHR enlacando no está habilitado para este proyecto.",
+    "search-for-provider": "Encontrar a mi Proveedor",
+    "reconnect": "Volver a conectar",
+    "connected": "Conectado",
+    "search": "Buscar",
+    "connect-fitbit-intro": "Si ya tiene una cuenta Fitbit, puede compartir información desde allí. Para empezar, presione el siguiente botón para ingresar a su cuenta de Fitbit:",
+    "connect-fitbit-button": "Conectar a Fitbit",
+    "received-fitbit-data": "¡Estamos recibiendo su información de Fitbit!",
+    "downloading-data": "Descargando datos...",
+    "all-tasks-complete": "¡Todas las tareas terminadas!",
+    "view-all": "Ver todo",
+    "tasks": "Tareas",
+    "incomplete-tasks": "Tareas incompletas",
+    "completed-tasks": "Tareas completadas",
+    "overdue": "Vencidos",
+    "due-today": "Vence hoy",
+    "due-tomorrow": "Vence mañana",
+    "due": "Vence",
+    "due-in": "Vencido en",
+    "days": "dias",
+    "notifications": "Notificaciones",
+    "support": "Soporte",
+    "all-notifications": "Todas las notificaciones",
+    "steps": "Pasos",
+    "resting-heart-rate": "Ritmo cardíaco en reposo",
+    "distance-traveled": "Distancia Viajada",
+    "google-fit-share": "Toca para compartir datos de condición físic",
+    "devices": "Dispositivos",
+    "last-sync": "Última sincronización",
+    "completed": "Completadas",
+    "ehr-intro": "Sus registros de salud electrónicos son fuentes importantes de información. Pueden ayudar a investigadores hacer nuevos descubrimientos.",
+    "external-accounts-title-divider": " / ",
+    "external-accounts-title-providers": "Proveedores",
+    "external-accounts-title-health-plans": "Planes de Salud",
+    "external-accounts-title-devices": "Dispositivos",
```

Comment:
I wasn't sure what the capitalization rules should be here for `external-accounts-title-health-plans` and `external-accounts-title-devices`.  In the full title, they were not capitalized, but since they can appear by themselves now, I thought they should be.  If there is an issue with having them be capitalized when they might be included as the second or third heading, let me know and I will revisit this.